okhttp/src/main/java/okhttp3/Protocol.java

```diff
@@ -78,12 +85,15 @@ public static Protocol get(String protocol) throws IOException {
     if (protocol.equals(HTTP_1_1.protocol)) return HTTP_1_1;
     if (protocol.equals(HTTP_2.protocol)) return HTTP_2;
     if (protocol.equals(SPDY_3.protocol)) return SPDY_3;
+    if (protocol.equals(QUIC.protocol)) return QUIC;
```

Comment:
I wonder if we need this. This method is used for ALPN, and QUIC doesn't use ALPN. In particular, the string "quic" isn't used on the we wire anywhere, is it?

Reply:
It's useful as API

https://github.com/yschimke/oksocial/blob/master/src/main/kotlin/com/baulsupp/oksocial/commands/CommandLineClient.kt#L450

Reply:
Works for me.